src/ontology/bridge/collected-drosophila.owl

```diff
@@ -32,6 +32,6 @@
     </rdfs:comment>
     <owl:imports rdf:resource="http://purl.obolibrary.org/obo/uberon.owl"/>
     <owl:imports rdf:resource="http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-fbbt.owl"/>
-    <owl:imports rdf:resource="http://purl.obolibrary.org/obo/uberon/fbbt-nd.owl"/>
+    <owl:imports rdf:resource="http://purl.obolibrary.org/obo/fbbt.owl"/>
```

Comment:
> "which we update here to use the imports/local-fbbt.owl mirror instead."

This is not immediately clear from this PR here - is this because the catalog rewires http://purl.obolibrary.org/obo/fbbt.owl to imports/local-fbbt.owl?

Reply:
Yes – as for all the other foreign ontologies that are imported in the various `collected-*.owl` files. The pattern is always the same: the `collected-*.owl` file mentions the foreign ontology by its “normal” IRI, and the catalog redirects it to the `imports/local-*.owl` copy.

`collected-drosophila.owl` was the only collected ontology to deviate from that pattern, because it imported that `fbbt-nd.owl` instead. So here I just made `collected-drosophila.owl` follow the same pattern as all the others.

(This pattern is one of the thing that I will amend when I’ll fix the composite pipeline, by the way. I think it is needlessly confusing, because you need to look up into the catalog to actually know _which_ file is being used. But I didn’t want to do that just yet, not in this PR which is, err, long enough already?)

Reply:
Thanks!